datafusion/core/tests/physical_optimizer/enforce_distribution.rs

```diff
@@ -371,46 +371,95 @@ macro_rules! plans_matches_expected {
     }
 }
 
+fn test_suite_default_config_options() -> ConfigOptions {
+    let mut config = ConfigOptions::new();
+
+    // By default, will not repartition / resort data if it is already sorted.
+    config.optimizer.prefer_existing_sort = false;
+
+    // By default, will attempt to convert Union to Interleave.
+    config.optimizer.prefer_existing_union = false;
+
+    // By default, will not repartition file scans.
+    config.optimizer.repartition_file_scans = false;
+    config.optimizer.repartition_file_min_size = 1024;
+
+    // By default, set query execution concurrency to 10.
+    config.execution.target_partitions = 10;
+
+    // Use a small batch size, to trigger RoundRobin in tests
+    config.execution.batch_size = 1;
```

Comment:
This matches main: https://github.com/apache/datafusion/blob/c61e7e597f80a3abd5015c996dc896a084112ab6/datafusion/core/tests/physical_optimizer/enforce_distribution.rs#L412-L413